Training5/src/task1/src/components/TodoEdit.js

```diff
@@ -46,14 +46,16 @@ const EditForm = styled.form`
   }
 `;
 export default function TodoEdit() {
-  const todo = useSelector((state) => state.todo.editTodo);
+  const id = useSelector((state) => state.todo.editTodoId);
```

Comment:
@phuongluongZigvy you can write ONE selector getCurrentEditTodoId = (state) => todo, you don't need get todos in the selector at this component, if you get like this, any todo in list changes (update or remove), this component will be re-rendered